protocol/x/vest/types/keys.go

```diff
@@ -15,6 +15,12 @@ const (
 
 	// VestEntryKeyPrefix is the prefix used when storing a VestEntry in the state.
 	VestEntryKeyPrefix = "vest_entry"
+
+	// CommunityTreasury defines the root string for community treasury module account.
+	CommunityTreasury = "community_treasury"
```

Comment:
Should we suffix these with `AccountName` to be consistent with other module account names?

Reply:
Sounds good!